CriteoMoPubAdapter/Tests/CriteoMoPubAdapterTests/Info.plist

```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>BNDL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.0</string>
+	<string>4.0.0-alpha1</string>
```

Comment:
Shouldn't this be 4.0.0.0-alpha1 ? Or do you want to to have a unique version for all SDK+adapters ?

Reply:
With that merge there will be only one package with one version

Reply:
Also means one changelog, if you want to amend an adapter you have to push the Sdk as well

Reply:
TODO: Changelog 😅